ts/src/tokocrypto.ts

```diff
@@ -1602,6 +1602,9 @@ export default class tokocrypto extends Exchange {
         //
         if (uppercaseType === 'MARKET') {
             const quoteOrderQty = this.safeValue (this.options, 'quoteOrderQty', true);
+            if (this.options['createMarketBuyOrderRequiresPrice'] && side === 'buy') {
+                priceIsRequired = true;
```

Comment:
in this case we should throw a meaningful message to the user ` throw new errors.InvalidOrder(this.id + ' createOrder() requires price argument for market buy orders on spot markets to calculate the total amount to spend (amount * price), alternatively set the createMarketBuyOrderRequiresPrice option to false and pass in the cost to spend into the amount parameter');`

Reply:
and usually we also check the `cost` in `params`

Reply:
check a few lines below, cost is checked for